src/pod-security-policy/capabilities/src.rego

```diff
@@ -35,8 +35,9 @@ has_disallowed_capabilities(container) {
 }
 
 missing_drop_capabilities(container) {
-  must_drop := {c | c := input.parameters.requiredDropCapabilities[_]}
```

Comment:
thanks for the PR! can you update the `template.yaml` with this change too please?

Reply:
done